language-server/modules/langserver-compiler/src/test/java/org/ballerinalang/langserver/compiler/workspace/WorkspaceDocumentManagerImplTest.java

```diff
@@ -0,0 +1,145 @@
+package org.ballerinalang.langserver.compiler.workspace;
```

Comment:
Missing License header

Reply:
Will fix it

Reply:
Fixed with https://github.com/ballerina-platform/ballerina-lang/pull/9836/commits/b6cca5194c5bcef685f1c8104c5ab992ee8f9dfa